lib/container_server_unsupported.go

```diff
@@ -11,3 +14,8 @@ func (c *ContainerServer) addSandboxPlatform(sb *sandbox.Sandbox) {
 func (c *ContainerServer) removeSandboxPlatform(sb *sandbox.Sandbox) {
 	// nothin' doin'
 }
+
+func (c *ContainerServer) getContainerStats(ctr *oci.Container, previousStats *ContainerStats) (*ContainerStats, error) {
+	// nothin' doin'
+	return nil, nil
```

Comment:
I think we need to return `nil, error.New("container stats not supported")` or some such.  Otherwise, it will get consumed [here][1] and panic [here][2].  If we return an error non-Linux users will at least have something prettier to look at until they work up a real implementation.

[1]: https://github.com/kubernetes-incubator/cri-o/blob/0e0c820f0c0a4220a0462d2b89bd18a9f0012e2f/server/container_stats.go#L47-L52
[2]: https://github.com/kubernetes-incubator/cri-o/blob/0e0c820f0c0a4220a0462d2b89bd18a9f0012e2f/server/container_stats.go#L22

Reply:
done